pennylane/drawer/tape_text.py

```diff
@@ -36,27 +64,93 @@ def _add_grouping_symbols(op, layer_str, wire_map):
     return layer_str
 
 
-def _add_op(op, layer_str, wire_map, decimals, cache):
+def _add_cond_grouping_symbols(op, layer_str, config):
+    """Adds symbols indicating the extent of a given object for conditional
+    operators"""
+    n_wires = len(config.wire_map)
+
+    mapped_wires = [config.wire_map[w] for w in op.wires]
+    mapped_bits = [config.bit_map[m] for m in op.meas_val.measurements]
+    max_w = max(mapped_wires)
+    max_b = max(mapped_bits) + n_wires
+
+    ctrl_symbol = "╩" if config.cur_layer != config.final_cond_layers[max(mapped_bits)] else "╝"
+    layer_str[max_b] = "═" + ctrl_symbol
+
+    for w in range(max_w + 1, max(config.wire_map.values()) + 1):
+        layer_str[w] = "─║"
+
+    for b in range(n_wires, max_b):
```

Comment:
Yeezy, Yeezy, 🐻 what's good? 😂😂👍🏻👍🏻 It's your boy Max B 😎, what's going on? 😉 Just checking ✔✔ in on you 😁 Appreciate the love and support ❤❤ The wave 🌊 is here 💪🏻 You a wavey 🌊 dude